packages/gatsby-source-contentful/src/__tests__/__snapshots__/normalize.js.snap

```diff
@@ -4665,18 +4466,22 @@ Unsere Lemnos Produkte werden sorgfältig von unseren Handwerkern fein geschliff
     Object {
       "children": Array [
         "uuid-from-gatsby",
+        "uuid-from-gatsby",
```

Comment:
@pieh where are these coming from? any idea?

Reply:
I don't know why we have new child here, but changing the mocked function from `createNodeId.mockReturnValue(`uuid-from-gatsby`)` to some pass-through function so we can get some ids to identify other nodes could be good idea here at least to identify what those children are and if they should be here or not